website/project/views/contributor.py

```diff
@@ -505,6 +507,29 @@ def send_claim_email(email, user, node, notify=True, throttle=24 * 3600):
     return to_addr
 
 
+@contributor_added.connect
+def notify_contributor(node, contributor, throttle=24 * 3600):
```

Comment:
The throttle should be configurable in `website.settings`--QA wouldn't be too happy if they had to wait 24 hours just to test this =).